packages/application-shell/src/components/quick-access/quick-access.js

```diff
@@ -79,7 +79,10 @@ class QuickAccess extends React.Component {
     onChangeProjectDataLocale: PropTypes.func,
     pimSearchProductIds: PropTypes.func.isRequired,
     getPimSearchStatus: PropTypes.func.isRequired,
-    useFullRedirectsForLinks: PropTypes.bool,
+    environment: PropTypes.shape({
+      useFullRedirectsForLinks: PropTypes.oneOf([true, false, 'true', 'false'])
```

Comment:
I guess, like we all, I find this a bit gnarly. Any component relying on this prop will have a quite bloated prop-type eventually. I think it's fine for now but somehow the application context provider (I presume) could also normalise these boolean flags coming in as strings.

Reply:
Yeah, but in the app shell we usually don't read from the context. I can coerce the values in the top component though...let me try.

Reply:
Yeah. That's what I meant to suggest. Thanks for trying!